src/bot.py

```diff
@@ -249,8 +253,12 @@ async def message_callback(self, room: MatrixRoom, event: RoomMessageText) -> No
                 or self.gpt_api_endpoint != "https://api.openai.com/v1/chat/completions"
             ):
                 n = self.chat_prog.match(content_body)
-                if n:
-                    prompt = n.group(1)
+                nd = content_body[:1] != '!'
+                if n or nd:
+                    if n:
```

Comment:
We need a symbol to trigger the bot. In your case, bot will respond even without `!`.